cmd/downloadTurncat.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
```

Comment:
Istio can get away with a single script because they install only a single utility, but we have two. What if we provide only a single `getstunner,sh` script but it downloads both `stunnerctl` and `turncat`?

Reply:
Done in 3405d04